File: Strumento.java
class Strumento extends Carta{
    private String azione;

    public Strumento(String a, String nome, String immagine){
        super(nome, immagine, "strumento");
        this.azione=a;
    }

    public String getAzione(){
        return this.azione;
    }
}

File: PockemonTCGp_Grafica/PockemonTCGp_Grafica/Strumento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PockemonTCGp_Grafica
{
    public class Strumento : Carta
    {
        private string azione;

        public Strumento(string a, string nome, string immagine) : base(nome, immagine, "strumento")
        {
            this.azione = a;
        }

        public string GetAzione()
        {
            return this.azione;
        }
    }
} //---Dzyubanov---
